app/code/Bulbulatory/Recomendations/Helper/Email.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Bulbulatory\Recomendations\Helper;
+
+use Magento\Framework\UrlInterface;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\Translate\Inline\StateInterface;
+use Magento\Framework\Escaper;
+use Magento\Framework\Mail\Template\TransportBuilder;
+
+class Email extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    protected $inlineTranslation;
+    protected $escaper;
+    protected $transportBuilder;
+    protected $urlInterface;
+    protected $logger;
+
+    public function __construct(
+        Context $context,
+        StateInterface $inlineTranslation,
+        Escaper $escaper,
+        TransportBuilder $transportBuilder,
+        UrlInterface $urlInterface
+    ) {
+        parent::__construct($context);
+        $this->inlineTranslation = $inlineTranslation;
+        $this->escaper = $escaper;
+        $this->transportBuilder = $transportBuilder;
+        $this->urlInterface = $urlInterface;
+        $this->logger = $context->getLogger();
+    }
+
+    public function sendEmail($email, $hash)
+    {
+
+        $templateId = 'recommendation_email_template';
```

Comment:
Dodałaś pole w konfiguracji na email template, ale tutaj jest id templatki wpisany na sztywno, przez co zmiany w konfiguracji nie będą miały wpływu na działanie. W takim wypadku albo usuń opcję wyboru templatki z konfiguracji (skoro i tak to nic nie wnosi) albo dodaj pobieranie templatki z konfiguracji.